library/elixir

```diff
@@ -33,12 +33,12 @@ Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
 GitCommit: 4122b4840bd762d1434424e1ec693929b0198c98
 Directory: 1.8/alpine
 
-Tags: 1.8.2-otp-22, 1.8-otp-22, otp-22
+Tags: 1.8.2-otp-22, 1.8-otp-22
 Architectures: amd64, arm32v7, arm64v8, i386, s390x, ppc64le
 GitCommit: 6dc5ffd3b4c2915096887b45ba8e71d391ce2398
 Directory: 1.8/otp-22
 
-Tags: 1.8.2-otp-22-alpine, 1.8-otp-22-alpine, otp-22-alpine
+Tags: 1.8.2-otp-22-alpine, 1.8-otp-22-alpine
```

Comment:
Are the `elixir:otp-22` and `elixir:otp-22-alpine` tags meant to be removed?

(just double checking that this is the desired outcome)

Reply:
I recheck the code, the 1.9 branch is the current latest branch, and it complie with otp 22. No need the otp-22-alpine, just the alpine is fine.
But the 1.8 is not the latest branch.
So I think the code is correct.